### DISCUSSION TITLE
Feature Request - Brush Name Tooltip on hover in pie menus

### DISCUSSION THREAD

**Discussion by richardsdev** [wojtryb/Shortcut-Composer#64] [Ideas]
Firstly, thanks and congratulations on this amazing plugin. After setting this up and playing with it for a few hours I am much more able to concentrate on the canvas instead of selecting things.

As a relative newbie, I still struggle to sometimes tell apart the brushes I have added to pie menus. Still building my library of "favourites" and "specials". So I am often iterating on brush settings, and have brushes with different version numbers on them. For easier brush identification it would be useful to see the name of the brush appear somewhere as I select them in the pie menu. I don't really mind how it is displayed, but I would have thought a tooltip or floating label of some kind would be most appropriate. Obviously that might not be needed for most artists, and would just look like clutter, but perhaps this could be an optional feature within the config dialogue?

**Comment by wojtryb** [comment]
Hi, and thanks for your feedback.

A simple label centered below the pie would be nice to have. It could appear when cursor is outside of the deadzone, and probably it should be possible to turn it off.

It shouldn't be that hard to do, but I never did that as it never felt essential. I'll keep this in mind.
